app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,74 @@
+<%= form_with(model: @recipe, url: recipes_path, local: true) do |form| %>
+  <div class = "mt-4 p-2">
+    <%= select_tag(:public, options_for_select([['Select Status', 'Status'], ['Public', 'Public'], ['Private', 'Private']], 'Status'),
+      class: 'form-select')
+    %>
+    <% if @recipe.errors.any? %>
+        <p class="text-danger mb-0">
+          <%= @recipe.errors[:status].first %>
+        </p>
+    <% end %>
+  </div>
+
+	<div class = "mt-2 p-2">
+		<%= form.text_field :name,
+        autofocus: true, 
+        autocomplete: 'name', 
+        placeholder: 'New inventory name', 
+        class: 'form-control' 
+    %>
+    <% if @recipe.errors.any? %>
+        <p class="text-danger mb-0">
+          <%= @recipe.errors[:name].first %>
+        </p>
+    <% end %>
+	</div>
+
+  <div class = "mt-2 p-2">
+    <%= form.number_field :preparation_time,
+        autofocus: true,
+        autocomplete: 'preparation_time',
+        placeholder: 'Preparation time in hour(s)',
+        class: 'form-control'
+    %>
+    <% if @recipe.errors.any? %>
+        <p class="text-danger mb-0">
+          <%= @recipe.errors[:preparation_time].first %>
+        </p>
+    <% end %>
+  </div>
+
+  <div class = "mt-2 p-2">
+    <%= form.number_field :cooking_time,
+        autofocus: true,
+        autocomplete: 'cooking_time',
+        placeholder: 'Cooking time in hour(s)',
+        class: 'form-control'
+    %>
+    <% if @recipe.errors.any? %>
+        <p class="text-danger mb-0">
+          <%= @recipe.errors[:cooking_time].first %>
+        </p>
+    <% end %>
+  </div>
+
+	<div class = "mt-2 p-2">
+		<%= form.text_area :description, rows: "5",
+        autofocus: true, 
+        autocomplete: 'description', 
+        placeholder: 'Description of the inventory', 
```

Comment:
- [ ] I suggest that the placeholder for this should be `Description of the recipe` instead of `Description of the inventory` since this form is for creating recipes and not inventory :blush: